curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/660f447efc0e722f016c1be0.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 660f447efc0e722f016c1be0
+title: Step 55
+challengeType: 0
+dashedName: step-55
+---
+
+# --description--
+
+The text has appeared again! This is because `"false"` is a string, which when evaluated to a boolean becomes `true`. `"false"` is considered a <dfn>truthy</dfn> because of this.
+
+Try changing the condition to an empty string `""`.
```

Comment:
I think we should expand on the defintion of truthy and falsey here

```suggestion
The text has appeared again! This is because `"false"` is a string, which when evaluated to a boolean becomes `true`. 

`"false"` is considered a <dfn>truthy</dfn> because of this.

A truthy value is a value that is considered true when encountered in a Boolean context. All values are truthy unless they are defined as `falsy`. Falsy values are values that are considered false when encountered in a Boolean context.

Here is a list of common falsy values in JavaScript:

- `false`
- `0`
- `""`
- `null`
- `undefined`
- `NaN`

You will learn more about some of those values later on in the curriculum.

Try changing the condition to an empty string `""`.
```